addons/sys_io/fnc_server.sqf

```diff
@@ -31,7 +31,7 @@ DFUNC(connectionFnc) = {
                         if ((missionNamespace getVariable [QGVAR(notConnectedTime), -15]) + 30 < time ) then {
                             GVAR(notConnectedTime) = time;
                             private _warning = format ["<t color='#FF8021'>WARNING!</t><br/> %1", localize LSTRING(acreNotConnected)];
-                            [[_warning, 1.5]] call CBA_fnc_notify;
+                            [[_warning, 2]] call CBA_fnc_notify;
```

Comment:
No, keep it at 1.5. 2 has trouble going off-screen, it's just too big.

Reply:
Merge issue...